--- hanhbuild	2021-10-20 10:31:01.911928219 +0700
+++ hanhbuild1	2021-10-22 16:09:13.590721556 +0700
@@ -79,6 +79,7 @@
    [ -f $search_dir/$binary ] && final_binary=1 
   done
  [ -z "$final_binary" ] && err "Binary $binary not found!"
+ unset final_binary
  done
 
  info "Finished binaries check"
@@ -95,6 +96,7 @@
    [ -f "$search_dir/$library" ] && final_lib=1
   done
   [ -z "$final_lib" ] && err "Library $library not found!"
+  unset final_lib
  done
 
  info "Finished libraries check"
